server/neptune/workflows/internal/terraform/state/workflow.go

```diff
@@ -89,6 +89,7 @@ type WorkflowResult struct {
 }
 
 type Workflow struct {
+	Mode     *terraform.WorkflowMode
```

Comment:
can you add a comment here for future readers explaining that mode dictates which jobs are populated